docs/README.md

```diff
@@ -261,6 +261,9 @@ The valid values for this option are:
    * When set to **`start`**:
       * If there is information in the “state file”, use the last recorded state. 
       * If the file is not present in the “state file”, start at the beginning. 
+   * When set to **`tail`**:
+      * Will read .log files starting at the beginning if created after the agent has started. This can result in large .log files being processed if copied into log directory. A WARN log will be printed if this happens.  
```

Comment:
Maybe reference the default behavior to refresh the reader's mind. It starts from the beginning of the file, but as opposed to what? Just a suggestion.

Reply:
👍